sensor_msgs/Image point_map
sensor_msgs/Image normal_map
sensor_msgs/Image mask
---
sensor_msgs/Image graspable_map